### ISSUE TITLE
Script can't find Perl dependencies

### ISSUE THREAD

**Issue by konradjurk** [neographophobic/autoMunkiImporter#7]
Hi, I can't get this to work on Mac OS 10.15.4 Catalina. Is it possible at all and if so, how might I be able to fix this problem? :) 

```
autoMunkiImporter $ sudo autoMunkiImporter.pl --force-install-dependencies
Password:
Term::UI will be removed from the Perl core distribution in the next major release. Please install it from CPAN. It is being used at /usr/local/autoMunkiImporter/autoMunkiImporter.pl, line 53.
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  295k  100  295k    0     0   618k      0 --:--:-- --:--:-- --:--:--  618k
Installing Date::Parse
Date::Parse is up to date. (2.32)

Installing Mail::Mailer
Mail::Mailer is up to date. (2.21)

Installing URI::Escape
URI::Escape is up to date. (3.31)

Installing URI::URL
URI::URL is up to date. (5.04)

Installing WWW::Mechanize
WWW::Mechanize is up to date. (1.96)

Attempted installation of required modules complete...
[ERROR] Required Perl Modules were not found (you can attempt to install them by running autoMunkiImporter.pl --install-dependencies ).
[ERROR]  - Please ensure that Date::Parse, Mail::Mailer, URI::Escape, URI::URL, and WWW::Mechanize are installed.
```

**Comment by neographophobic** [comment]
Hi Konrad,

I'm no longer working in this space, but will try to have a look at why this is failing and get a fix. That said it will take me a while to investigate.

In the mean time I suggest having a look at https://github.com/autopkg/autopkg/wiki/Getting-Started. It seems like the Munki community is primarily using Autopkg, which does similar things to what this script does, and has a far more active user community.

Cheers,
Adam

**Comment by konradjurk** [comment]
@neographophobic Thank you! I totally understand and also expected your recommendation for Autopkg, but I am extending the API of our corporate download portal for one of our customers who has your tool rolled out, so I don't have a choice in that regard.

Closing for now as I extracted the most interesting parts for my testing purposes into a separate Perl script which I can execute. In case I have a question, I might come back to you here if thats ok.
